Tasks/Common/AzureHelpers/AzureHelpers.psm1

```diff
@@ -1,4 +1,38 @@
-﻿Import-VstsLocStrings -LiteralPath $PSScriptRoot\module.json
-. $PSScriptRoot\PrivateFunctions.ps1
-. $PSScriptRoot\PublicFunctions.ps1
+﻿# Private module-scope variables.
```

Comment:
I rearranged this helper module for readability.

There is only one public function so it moved it directly into the psm1 file. I moved all of the private "import" functions into ImportFunctions.ps1 and "initialization" functions into InitializeFunctions.ps1